polyaxon/templates/api-deployment.yaml

```diff
@@ -128,7 +128,7 @@ spec:
           httpGet:
             path: /_health/
             port: {{ .Values.streams.service.internalPort }}
-            scheme: HTTP
+            scheme: {{ .Values.streams.service.scheme | default "HTTP" }}
```

Comment:
it should use `.Values.api.service.scheme` as well, there will be only one user facing service starting from v0.5

Reply:
changed to `.Values.api.service.scheme`